Dockerfile

```diff
@@ -22,9 +22,10 @@ RUN rm /tmp/cuchem_rapids_0.17.yml
 RUN source activate cuchem && python3 -m ipykernel install --user --name=cuchem
```

Comment:
A couple of the Dockerfile issues still persist, and this causes the container build to hang for a long time, even though it does complete. The issues include the need to remove an errant flag from conda clean and to change the way conda is invoked when a package is installed. I've pushed an updated version of the container [here](https://github.com/mlgill/cheminformatics/blob/rilango/prep-release/Dockerfile), which also contains the correct way to clean apt and conda (in the same RUN command) so that the size of the docker image layers stays small. Please use this file in its entirety unless there are specific issues that arise.